x-pack/plugins/ml/public/application/components/node_available_warning/hooks.ts

```diff
@@ -74,12 +74,12 @@ const defaultCloudInfo: CloudInfo = {
 
 export function useCloudCheck() {
   const { http } = useKibana().services;
-  const ml = useMemo(() => mlApiServicesProvider(new HttpService(http!)), [http]);
+  const mlApi = useMemo(() => mlApiProvider(new HttpService(http!)), [http]);
```

Comment:
Is there a reason for this specific code or could this be `const mlApi = useMlApi()`?